terraform/layer1-aws/variables.tf

```diff
@@ -5,6 +5,22 @@ variable "allowed_account_ids" {
   default     = []
 }
 
+variable "aws_account_password_policy" {
+  type = any
+  default = {
+    create                         = true
+    minimum_password_length        = "14" # Minimum length to require for user passwords
+    password_reuse_prevention      = "10" # The number of previous passwords that users are prevented from reusing
+    require_lowercase_characters   = true # If true, password must contain at least 1 lowercase symbol
+    require_numbers                = true # If true, password must contain at least 1 number symbol
+    require_uppercase_characters   = true # If true, password must contain at least 1 uppercase symbol
+    require_symbols                = true # If true, password must contain at least 1 special symbol
+    allow_users_to_change_password = true # Whether to allow users to change their own password
+    max_password_age               = "90" # How many days user's password is valid
+    hard_expiry                    = true # Whether users are prevented from setting a new password after their password has expired 
```

Comment:
i think this is too strict option.
without using aws web console, you may not notice the warnings.
and resetting the password will require the participation of the root account, which may take time